backends/platform/sdl/posix/posix.cpp

```diff
@@ -141,6 +141,22 @@ Common::String OSystem_POSIX::getDefaultConfigFileName() {
 	return configFile;
 }
 
+void OSystem_POSIX::addSysArchivesToSearchSet(Common::SearchSet &s, int priority) {
+	const char *snap = getenv("SNAP");
+	if (snap) {
```

Comment:
It should still call `OSystem_SDL::addSysArchivesToSearchSet` even when `SNAP` is not set.